src/robot/conf/settings.py

```diff
@@ -18,6 +18,7 @@
 import time
 
 from robot import utils
+from robot.utils import is_string
```

Comment:
Other utils are used like `utils.function()` in this module. For consistency reasons it would be better to use also `utils.is_string()` instead of just `is_string()`.

Unfortunately different modules in our code base uses utils like `utils.is_string()` and `is_string()`. The reason is that we used to the former earlier and have been recently switching to latter style but haven't converted all modules to use the that style. Inconsistency between modules is obviously bad, but I consider inconsistency within a module is even worse.

What do you think, should we first convert all modules to use `is_string` style or would you be willing to change this PR to use `utils.is_string` style in modules that still use that style?